### PR TITLE
[zh] sync storage/volumes

### PR DESCRIPTION
**Pull request by 0xff-dev**
[zh] sync storage/volumes

### CODE REVIEW

**Comment by asa3311** [comment]
You can use the script(./scripts/lsync.sh content/zh-cn/docs/concepts/storage/volumes.md ) to compare the differences in files that need to be updated. Could you please update this content together.
image

File: content/zh-cn/docs/concepts/storage/volumes.md [hunk 1]
```diff
@@ -510,14 +500,17 @@ keyed with `log_level`.
...
+
 * A container using a ConfigMap as a [`subPath`](#using-subpath) volume mount will not
   receive ConfigMap updates.
 
 * Text data is exposed as files using the UTF-8 character encoding. For other character encodings, use `binaryData`.
 -->
 {{< note >}}
 * 在使用 [ConfigMap](/zh-cn/docs/tasks/configure-pod-container/configure-pod-configmap/) 之前你首先要创建它。
-* 容器以 [subPath](#using-subpath) 卷挂载方式使用 ConfigMap 时，将无法接收 ConfigMap 的更新。
+* ConfigMap 总是以只读的方式挂载。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/storage/volumes.md:512]: 1. 反引号的内容不要翻译，大多是代码、属性名称之类的文字；
2. `readOnly` 是一种挂载模式，不是一种方式。

```suggestion
* ConfigMap 总是以 `readOnly` 的模式挂载。
```

File: content/zh-cn/docs/concepts/storage/volumes.md [hunk 2]
```diff
@@ -1524,17 +1517,16 @@ API 服务器上，然后以文件的形式挂载到 Pod 中，无需直接与 K
...
-<!--
-A container using a Secret as a [`subPath`](#using-subpath) volume mount will not
+* A Secret is always mounted as `readOnly`.
+
+* A container using a Secret as a [`subPath`](#using-subpath) volume mount will not
 receive Secret updates.
 -->
-容器以 [`subPath`](#using-subpath) 卷挂载方式挂载 Secret 时，将感知不到 Secret 的更新。
+* 使用前你必须在 Kubernetes API 中创建 Secret。
+* Secret 总是以只读的方式挂载
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/storage/volumes.md:1528]: 同前。

另外，结尾的句号不要丢，保持一致。

**Review by my-git9 — COMMENTED** [review]
This change conflicts with a previous PR: https://github.com/kubernetes/website/pull/41346

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**